Corona-Warn-App/config/detekt.yml

```diff
@@ -100,7 +100,7 @@ complexity:
     ignoreStringsRegex: '$^'
   TooManyFunctions:
     active: true
-    excludes: ['**/test/**', '**/androidTest/**', '**/*.Test.kt', '**/*.Spec.kt', '**/*.Spek.kt', '**/LocalData.kt', '**/formatter/*Helper.kt']
+    excludes: ['**/test/**', '**/androidTest/**', '**/*.Test.kt', '**/*.Spec.kt', '**/*.Spek.kt', '**/LocalData.kt', '**/formatter/*Helper.kt', '**/worker/BackgroundWorkScheduler.kt']
```

Comment:
Why do we need this here? Can we split up the scheduler code?

Reply:
@AlexanderAlferov What are your thoughts on this?

Reply:
@AlexanderAlferov  could you please comment here. We have to close this PR tonight (CC: @jakobmoellersap )